app/Http/Controllers/PartyController.php

```diff
@@ -87,10 +87,9 @@ public function index($group_id = null)
             $upcoming_events_in_area = null;
         } else {
             $upcoming_events = Party::upcomingEvents()->where('users_groups.user', Auth::user()->id)
-                ->take(3)
                 ->get();
 
-            $past_events = Party::UsersPastEvents([auth()->id()])->paginate(10);
+            $past_events = Party::UsersPastEvents([auth()->id()])->get();
```

Comment:
Could we also sort by event_date, most recent first?

Reply:
It turns out we are sorting on the query, but I think we lose this when putting items in to the store?  Have put a sort at the point of display for now, but probably a more efficient way of doing this.

Reply:
Will discuss at retro.